### PR TITLE
Add flag allowCrossFlow()

### PR DESCRIPTION
**Pull request by totto82**
This PR adds support for item 10 in welspecs where a flag is used to
determine whether the well should allow cross flow or not.

A test is added to check default behavior.

### CODE REVIEW

File: opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -1272,7 +1272,12 @@ namespace Opm {
         if (refDepthItem->hasValue(0))
             refDepth.setValue( refDepthItem->getSIDouble(0));
 
-        well = std::make_shared<Well>(wellName, m_grid , headI, headJ, refDepth, preferredPhase, m_timeMap , timeStep, wellCompletionOrder);
+        bool allowCrossFlow = true;
+        const std::string& allowCrossFlowStr = record->getItem("CROSSFLOW")->getTrimmedString(0);
```
Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: For new code I am trying to push the use of the build constants:
```
record->getItem<ParserKeywords::WELSPECS::CROSSFLOW>()->getTrimmedString(0);
```

**Comment by joakim-hove** [comment]
This looks fine - with test and all! 

Will merge as soon as the minor wish regarding the use of the auto generated class `ParserKeywords::WELSPECS` has been updaed.

**Comment by totto82** [comment]
@joakim-hove  Have done what you suggested.

**Comment by joakim-hove** [comment]
> Have done what you suggested.

Thank you - merging!

**Pull request merged**